### PR TITLE
add hospitalizations_last_n_days function #78

### PR DESCRIPTION
**Pull request by wankiewiczm-smh**
This merge request contains the code for issue #78 

If you have any suggestions on improving the function or the documentation (a different name or additional unit tests), please let me know!

### CODE REVIEW

File: R/hospitalizations_last_n_days.R [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#'   `n_hospitalizations`, which represents the number of previous
+#'   hospitalizations in the given window. The values in `n_hospitalizations`
+#'   will be 0, 1, or 2+.
+#'
+#' @export
+#'
+#' @examples
+#' # Default n_days of 30:
+#' \dontrun{
+#' derive_n_hosp_prev(cohort)
```
Comment (marchenaromk) [inline R/hospitalizations_last_n_days.R:42]: `derive_n_hosp_prev(cohort)` should be changed to `hospitalizations_last_n_days(cohort)`

File: R/hospitalizations_last_n_days.R [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+#' }
+#'
+#' # User-input window of 182 days (6 months)
+#' \dontrun{
+#' derive_n_hosp_prev(cohort, n_days = 182)
```
Comment (marchenaromk) [inline R/hospitalizations_last_n_days.R:47]: `derive_n_hosp_prev(cohort,n_days = 182)` should be changed to` hospitalizations_last_n_days(cohort,n_days = 182)`

**Review by marchenaromk — COMMENTED** [review]
Function works on my end 👍

File: R/hospitalizations_last_n_days.R (hunk 2 above)
Comment (wankiewiczm-smh) [reply R/hospitalizations_last_n_days.R:47]: Great catches, updated now. Thanks Kayley!

**Review by marchenaromk — APPROVED** [review]
👍

**Pull request merged**